vendor/github.com/netobserv/flowlogs-pipeline/pkg/confgen/flowlogs2metrics_config.go

```diff
@@ -78,6 +83,8 @@ func (cg *ConfGen) GenerateTruncatedConfig() []config.StageParam {
 			parameters[i] = config.NewTransformNetworkParams("transform_network", *cg.config.Transform.Network)
 		case "extract_aggregate":
 			parameters[i] = config.NewAggregateParams("extract_aggregate", cg.aggregateDefinitions)
+		case "extract_timebased":
+			parameters[i] = config.NewTimbasedParams("extract_timebased", cg.timebasedTopKs)
```

Comment:
Misspelling: NewTimebasedParams

Reply:
this is in vendors, so that's actually in FLP repo